### ISSUE TITLE
hostname is bytes type instead of str

### ISSUE THREAD

**Issue by bjmb** [eventlet/eventlet#427]
```python
from eventlet import monkey_patch
monkey_patch()

import socket

host = socket.getnameinfo(("127.0.0.1", 0), 0)[0]
assert isinstance(host, str)
```
The assertion fails in python3 but it's fine if we remove the `monkey_patch`. Looks like [this string](https://github.com/eventlet/eventlet/blob/master/eventlet/support/dns/name.py#L486) is returned but according to the socket API a string should be returned.

**Comment by temoto** [comment]
@bjmb Hello. Thanks for report.

Please, try again with this patch. `pip install https://github.com/eventlet/eventlet/archive/9e8c431fe387f37669d8a42f3bb70e4077dea571.zip`

**Comment by bjmb** [comment]
@temoto , yes it's working as expected with that patch

**Comment by temoto** [comment]
Thanks for testing.
It's merged in master 9e8c431fe387f37669d8a42f3bb70e4077dea571

Have fun.
